### PR TITLE
Replace 'search' text with search icon

### PR DESCRIPTION
**Pull request by geoffreyyip**
I rule and have completed some work on Case Manager that's ready for review!

I changed the submit tag to a search glyphicon 

Before:
screen shot 2018-02-25 at 7 31 24 pm

After:
screen shot 2018-02-25 at 7 27 08 pm

### CODE REVIEW

**Comment by lomky** [comment]
This kills the system tests :scream: I think if you whack-a-mole the system tests CircleCI has failing, it should work again.

**Comment by geoffreyyip** [comment]
Thanks @lomky, I'll add it to my todo list

**Comment by colinxfleming** [comment]
@geoffreyyip hey, I've been trying to dig up the source issue this is from and couldn't find it in a bit of searching -- would you mind linking it when you get a minute?

**Comment by lomky** [comment]
@colinxfleming It may or may not have an issue - it's noted as a todo in the code, see master branch `app/views/dashboards/_search_form.html.erb` line 4

**Comment by geoffreyyip** [comment]
@colinxfleming, @lomky's right here. There's no issue for it. I saw a TODO in the comment, thought it'd be good practice, and gave it a go.

@lomky, test pass now.

File: app/views/dashboards/_search_form.html.erb [hunk 1]
```diff
@@ -1,10 +1,11 @@
 <div id="search_form" class="margin-bottom">
   <h1 class="border-bottom title">Build your call list</h1>
 
-  <!-- TODO change submit tag to a search glyphicon -->
   <%= bootstrap_form_tag url: '/search', remote: true, layout: :inline do |p| %>
     <%= p.text_field :search, placeholder: 'Name (Susan Smith) or Phone (555-555-5555)', hide_label: true, class: 'search_form_input' %>
-    <%= p.submit 'Search', class: 'btn btn-primary' %>
+    <%= p.button :class => "btn btn-primary", title: "Search" do %>
```
Comment (colinxfleming) [inline app/views/dashboards/_search_form.html.erb]: I didn't know title attributes did this! dooooope

**Comment by colinxfleming** [comment]
Thanks @geoffreyyip ! appreciate your flagging this and taking care of it!

**Pull request merged**